yt_dlp/extractor/soundcloud.py

```diff
@@ -317,21 +305,39 @@ def _download_json(self, *args, **kwargs):
                 raise
 
     def _real_initialize(self):
-        self._CLIENT_ID = self._downloader.cache.load('soundcloud', 'client_id') or "T5R4kgWS2PRf6lzLyIravUMnKlbIxQag"  # 'EXLwg5lHTO2dslU5EePe3xkw0m1h86Cd' # 'YUKXoArFcqrlQn9tfNHvvyfnDISj04zk'
+        self._CLIENT_ID = self._downloader.cache.load('soundcloud', 'client_id') or "fXuVKzsVXlc6tzniWWS31etd7VHWFUuN"  # persistent `client_id`
         self._login()
 
     _USER_AGENT = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.105 Safari/537.36"
     _API_AUTH_QUERY_TEMPLATE = '?client_id=%s'
     _API_AUTH_URL_PW = 'https://api-auth.soundcloud.com/web-auth/sign-in/password%s'
+    _API_VERIFY_AUTH_TOKEN = 'https://api-auth.soundcloud.com/connect/session%s'
     _access_token = None
     _HEADERS = {}
     _NETRC_MACHINE = 'soundcloud'
+    _reserved_usernames = ['token', 'auth', 'oauth', 'bearer', 'authorization']
 
     def _login(self):
         username, password = self._get_login_info()
         if username is None:
             return
 
+        if username in self._reserved_usernames:
```

Comment:
If username is not in `oauth`, explain to the user that login is broken and only `-u oauth -p <TOKEN>` can be used